demo_policy/e2e.bats

```diff
@@ -1,5 +1,58 @@
 #!/usr/bin/env bats
 
+@test "should allow pod creation in default namespace" {
+    run kwctl run annotated-policy.wasm -r ./test_data/no_privileged_containers.json --settings-json '{"testScenario": "can-i-success"}' --replay-host-capabilities-interactions ./test_data/sessions/can-i-success.yml --allow-context-aware
+    echo "output = ${output}"
+    [ "$status" -eq 0 ]
+    [ $(expr "$output" : '.*allowed.*true') -ne 0 ]
+    [[ "$output" =~ '"allowed":true' ]]
+}
+
+@test "should deny node deletion" {
+    run kwctl run annotated-policy.wasm \
+        -r ./test_data/no_privileged_containers.json \
+        --settings-json '{"testScenario": "can-i-failure"}' \
+        --replay-host-capabilities-interactions ./test_data/sessions/can-i-failure.yml \
+        --allow-context-aware
+    echo "output = ${output}"
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ '"allowed":false' ]]
+}
+
+@test "should return pods when listing all resources with label selector" {
```

Comment:
`should be able to get all kubernetes resources by label` or something like that